common/data_guide.md

```diff
@@ -70,7 +70,7 @@ To learn more about FHIR, bulk FHIR specifications, or the  Beneficiary FHIR Dat
 * [Bulk FHIR specification](http://build.fhir.org/ig/HL7/VhDir/bulk-data.html){:target="_blank"}
 * [Beneficiary FHIR Data Server (BFD)/ Blue Button API](https://bluebutton.cms.gov/developers/){:target="_blank"}
 * [Beneficiary FHIR Data Server (BFD)/ Blue Button Implementation Guide](https://bluebutton.cms.gov/assets/ig/index.html){:target="_blank"}
-* [Intro to JSON Format](http://json.org){:target="_blank"} [and] [http://ndjson.org/](http://ndjson.org/){:target="_blank"}
+* [Intro to JSON Format](http://json.org){:target="_blank"} [and] [ndjson-spec](https://github.com/ndjson/ndjson-spec/){:target="_blank"}
```

Comment:
Can you sneak in a content change so the bullet is formatted as
* Intro to the [JSON Format](http://json.org) and [NDJSON](https://github.com/ndjson/ndjson-spec/)

?
```suggestion
* Intro to the [JSON Format](http://json.org){:target="_blank"} and [NDJSON](https://github.com/ndjson/ndjson-spec/){:target="_blank"}
```